test/modules/ModuleLib/TestResultReportingClient.swagger.yaml

```diff
@@ -23,14 +23,14 @@ paths:
         description: "Test result details"
         required: true
         schema:
-          $ref: "#/definitions/TestOperationResult"
+          $ref: "#/definitions/ClientTestOperationResult"
```

Comment:
Can you make the changes to `description` on the top of the file to make it explicit that this is a writer API which is shared between Analyzer Module and TRC Module for a reporting client please?